cachito/workers/tasks/rubygems.py

```diff
@@ -16,3 +23,61 @@ def cleanup_rubygems_request(request_id):
         "username": get_rubygems_nexus_username(request_id),
     }
     nexus.execute_script("rubygems_cleanup", payload)
+
+
+def _get_config_file_for_given_package(
+    dependencies, bundle_dir, package_source_dir, rubygems_hosted_url
+):
+    """
+    Get Bundler config file.
+
+    Returns a Bundler config file with a mirror set for RubyGems dependencies pointing to
+    `rubygems_hosted_repo` URL. All GIT dependencies are configured to be replaced by local git
+     repos.
+
+    :param dependencies: an array of dependencies (dictionaries) with keys
+        "name": package name,
+        "path": an absolute path to a locally downloaded git repo,
+        "kind": dependency kind
+    :param bundle_dir: and absolute path to a bundle root directory
```

Comment:
Could you mention here that `bundle_dir` is the root of the Cachito bundle? At first I thought it was a Bundler bundle and was confused what was the difference between this and `package_source_dir`